### ISSUE TITLE
[Question] Where can I find seed files?

### ISSUE THREAD

**Issue by jolespin** [AuReMe/metage2metabo#48]
I'm trying to test out m2m workflow but it says I need to find a seed file.  Where I can find these? Is there a general one I can use or do they all have to be domain-specific? For example, can I use one for human oral microbiomes and then use the same one for marine microbiomes? If not, where can I find different ones?

**Comment by ArnaudBelcour** [comment]
Hi @jolespin,

The seed file corresponds to the metabolites available for the organisms in the environment (it is their growth medium). So there is no general seed file, each seed file will change according to the environment you study.

I do not think it could be relevant to use the same for microbiota in such different environments.

We create the seed file from literature and database. For example, for the human gut microbiota, we based your seed on the [EU average diet from the VMH database](https://www.vmh.life/#diet/EU%20average). The one we used in the article of Metage2Metabo for the human gut microbiota is available [here](https://github.com/AuReMe/metage2metabo/blob/master/article_data/gut_microbiota/seeds_gut_final.sbml).
Another example is the analysis of the microbiome of the brown algal _Ectocarpus siliculosus_, in which the authors used the Provasoli medium (you have the description of the [medium in this article](https://onlinelibrary.wiley.com/doi/10.1111/tpj.12627)).

To create a seed file, my advices are:
- (1) talk with biologists working on the environment, if possible.
- (2) search for other article working on similar communities to see if there are not already some works done on describing the seeds.
-  (3) search for articles describing the metabolites available in specific environment (for example with metabolomics experiments).
-  (4) look at databases associated with your environment.

The article by [Marinos et al. (2020)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7428157/) can help you see a roadmap to create the seed file. But there is a difference: with Metage2Metabo, we used a boolean abstraction of the metabolism, so we do not rely on the stochiometry of the reaction, so we do not need the concentration of input metabolites.
